### PR TITLE
Benp/break out tests

### PR DESCRIPTION
**Pull request by benpatterson**
@jzoldak @jibsheet 

This will help when reading packer output. Currently, there is just a general 'validate build environment' step that takes a very long time. Here, we'll provide a few more hints to the end-user, to get a sense of which step is being run. It also makes reading failures easier, since it's very clear which particular class of smoketests failed.

### CODE REVIEW

**Comment by jibsheet** [comment]
I like it :+1: Although I'd want @jzoldak 's thoughts as well since he's more familiar with the testing than I am (the ansible change makes sense).

File: playbooks/roles/test_build_server/files/test-development-environment.sh [hunk 1]
```diff
@@ -20,28 +20,46 @@ cd edx-platform-clone
...
+        # Generate some coverage reports
+        paver coverage
 
-# Run some of the javascript unit tests
-paver test_js_run -s xmodule
+        # Run some of the djangoapp unit tests
+        paver test_system -t lms/djangoapps/courseware/tests/tests.py
+        paver test_system -t cms/djangoapps/course_creators/tests/test_views.py
+        ;;
+    "js")
```
Comment (jzoldak) [inline playbooks/roles/test_build_server/files/test-development-environment.sh:38]: nit: this could use an empty line above

**Comment by jzoldak** [comment]
I like it!
:+1:

**Comment by benpatterson** [comment]
Updated with Jay's suggestion. Merging.

**Pull request merged**